src/components/LoginCard/index.jsx

```diff
@@ -1,50 +1,56 @@
 import React, { useState } from 'react';
 import { Link, useHistory } from 'react-router-dom';
 
-import { Button, Card, Form, Input, Space, Typography } from 'antd';
+import { Button, Card, Form, Input, Modal, Space, Typography } from 'antd';
 
 import { GoogleCircleFilled, LockOutlined, UserOutlined } from '@ant-design/icons';
 
-import './LoginCard.scss';
-import { useAuth } from 'services/AuthProvider';
+import { useAuth } from 'hooks/Hooks';
 import { googleProvider } from 'services/FirebaseService';
+import routes from 'constants/routes';
+
+import './styles.scss';
 
 const { Text } = Typography;
 
 function LoginCard() {
   const { emailLogin, googleLogin } = useAuth();
-  // eslint-disable-next-line no-unused-vars
-  const [loading, setLoading] = useState(false);
+  const [visible, setVisible] = useState(false);
```

Comment:
for booleans, lets add a prefix like `is` and `has`. for this case, lets rename this to `isVisible`. 

also, let's always make our boolean names "positive". 

more info here. https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
ah, lets also specify which is visible. so for this case, it should be `isModalVisible`